tests/bearlib/aspects/DecoratorsTest.py

```diff
@@ -16,15 +17,28 @@ class RunDecoratedBear(LocalBear):
         remove_unreachable_code=get_aspect('UnreachableCode'),
         minimum_clone_tokens=get_aspect('Clone').min_clone_tokens,
     )
+    @map_ambiguous_setting_to_aspect(
+        use_spaces=(get_aspect('Indentation').indent_type,
+                    [('space', True), ('tab', False)]),
+    )
     def run(self,
             remove_unreachable_code: bool = False,
             minimum_clone_tokens: int = 10,
+            use_spaces: bool = True,
             ):
-        return [remove_unreachable_code, minimum_clone_tokens]
+        return [('remove_unreachable_code', remove_unreachable_code),
+                ('minimum_clone_tokens', minimum_clone_tokens),
+                ('use_spaces', use_spaces)]
 
 
 class MapSettingToAspectTest(unittest.TestCase):
 
+    def list_to_dict(self, tuple_list):
```

Comment:
this doesnt use `self` so it should be a module level function, not inside the class